src/component/addons/AddonForm/AddonForm.jsx

```diff
@@ -79,7 +79,7 @@ export const AddonForm = ({ editMode, provider, addon, fetch }) => {
         setErrors({ ...errors, events: undefined });
     };
 
-    const handleCancel = () => {
+    const onCancel = () => {
```

Comment:
There are some issues with using `history.goBack()`. It works fine if you always think that the user will get to this page by usual flow. But if you go to this page via a direct link, then the cancel button will suddently take you back to the page in the history where you came from. 

Try going directly here and hit cancel: https://unleash-frontend-git-fix-remove-toast-on-cancel-unleash-team.vercel.app/addons/create/slack

@olav @ykhedher WDYT? It is a low risk, but maybe it is better to use explicit paths?

Reply:
Agreed, paths would be better. :+1: